packages/snfoundry/contracts/src/test/TestContract.cairo

```diff
@@ -175,7 +175,7 @@ fn test_withdraw_functionality() {
     assert_eq!(new_balance, expected_balance, "Balance should be increased by the stake amount");
 
     // Increase the block_timestamp by 30 seconds
-    start_cheat_block_timestamp_global(get_block_timestamp() + 30);
+    start_cheat_block_timestamp_global(get_block_timestamp() + 60);
```

Comment:
Change comment L177

// Increase the block_timestamp by 30 seconds by 60 seconds